### PR TITLE
✨ Source Card

### PR DESCRIPTION
**Pull request by jasangill1**
image

### CODE REVIEW

File: next/src/components/SourceCard.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { FaGlobe } from 'react-icons/fa';
+
+interface SourceCardProps {
+  title: string;
+  link: string;
+  position: number;
+}
+
+const SourceCard: React.FC<SourceCardProps> = ({ title, link, position }) => {
```
Comment (asim-shrestha) [inline next/src/components/SourceCard.tsx:10]: `const SourceCard = ({ title, link, position }: SourceCardProps) => {`

File: next/src/components/SourceCard.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  return (
+    <div className={`flex items-center p-2 border max-w-fit bg-gray-400 border-gray-400 rounded-md mt-2 ${position === 1 ? 'bg-yellow-100' : 'bg-blue-100'}`}>
```
Comment (asim-shrestha) [inline next/src/components/SourceCard.tsx:12]: Lets use clsx for stuff like this

Comment (asim-shrestha) [reply next/src/components/SourceCard.tsx:12]: Also why are we positionally changing CSS?

Comment (jasangill1) [reply next/src/components/SourceCard.tsx:12]: was thinking maybe we want to show which one was the newest add not sure if we wanted this will readjust

**Comment by jasangill1** [comment]
image

**Comment by jasangill1** [comment]
image

**Comment by jasangill1** [comment]
https://github.com/reworkd/AgentGPT/assets/56135840/80c353f5-9a2e-4f41-982d-24f08347775b

**Comment by jasangill1** [comment]
image

**Comment by awtkns** [comment]
@jasangill1 how does it look on mobile?

File: next/src/components/console/ChatMessage.tsx [hunk 3]
```diff
@@ -31,6 +32,31 @@ const ChatMessage = ({ message }: { message: Message }) => {
     }
   };
 
+  const renderSourceCardsGrid = () => {
```
Comment (awtkns) [inline next/src/components/console/ChatMessage.tsx:35]: Might be nice to extract this into its own component!

Comment (jasangill1) [reply next/src/components/console/ChatMessage.tsx:35]: yes daddy !

**Comment by jasangill1** [comment]
already added

**Pull request closed without merging**